docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.3.0
+    container_name: audit_elasticsearch
+    ports:
+      - 9200:9200
+      - 9300:9300
+    environment:
+      # ES_JAVA_OPTS: "-Xmx256m -Xms256m"
```

Comment:
There are more settings recommended at https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html

environment:
      - cluster.name=docker-cluster
      - bootstrap.memory_lock=true
      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
    ulimits:
      memlock:
        soft: -1
        hard: -1
      nofile:
        soft: 65536
        hard: 65536
    mem_limit: 1g
    cap_add:
      - IPC_LOCK

Perhaps we should be deploying a cluster of two nodes? With only one node indexes will be in yellow state as default index creation is that 2 replicas are created.